changelog/i18n-fixes

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: fix
+
+Fixing a few i18n instances in the plugin
```

Comment:
Indeed. Fixed c82e40524677d906862d136f9cd405f47352b089

Reply:
Thank you, it does read better now ❤️ But I feel there's still scope to expand on what "i18n instances" means since internationalization is a broader term.